src/shared/components/Workspace/Queries/Query/query-component.less

```diff
@@ -0,0 +1,55 @@
+@import '../../../../lib.less';
+
+.query-component {
+  .cardlike();
+  background-color: #f5f6f7;
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  max-height: 100%;
+  & > .infinite-scroll {
+    overflow: hidden;
+    overflow-y: auto;
+  }
+  & > .header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    color: @text-color; // instead of default heading color
+  }
+  & > .header > .count {
+    font-size: 0.8rem;
+    font-weight: lighter;
+    flex-grow: 1;
+    margin-left: 1em;
+  }
+  & > .controls {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 1em;
+    flex-wrap: wrap;
+  }
+  & > .controls > .search {
+    width: 60%;
```

Comment:
Mixing ems, rems, pxs and percents... Also saw viewport units (vh / vw) elsewhere. Confusing to me

Reply:
these percentages are for taking up width on a relative flexbox, just to look nice on the control panel.

 vh and vw are only for components that are layouts, basically, I think I have used those sparingly.

pxs should not really ever be used, instead em I guess